DEPS

```diff
@@ -66,7 +66,7 @@ vars = {
   'ios_tools_revision': '69b7c1b160e7107a6a98d948363772dc9caea46f',
 
   # Checkout Android dependencies only on platforms where we build for Android targets.
-  'download_android_deps': 'host_os == "mac" or host_os == "linux"',
+  'download_android_deps': 'host_cpu == "x64" and (host_os == "mac" or host_os == "linux")',
```

Comment:
Add a Todo/bug to enable this on arm

Reply:
https://github.com/flutter/flutter/issues/60118  The completed engine work was related to getting the ARM simulator binaries built. @gw280 has the most context.  The remaining work is mostly infra and getting the Flutter tooling running natively.

There's another umbrella issue for deploying a Flutter desktop app natively on an M1 without Rosetta: https://github.com/flutter/flutter/issues/60113